presto-docs/src/main/sphinx/release/release-0.156.rst

```diff
@@ -0,0 +1,26 @@
+=============
+Release 0.156
+=============
+
+General Changes
+---------------
+
+* Add support for quantified comparison predicates: ``ALL``, ``ANY`` and ``SOME``.
```

Comment:
I'd organize the items in this order:

* bug fixes (i.e., at timezone issue)
* new features (i.e., filtered aggs, quantified comparisons, new function, checksum improvements)
* performance improvements (i.e.. group by optimization, default task concurrency)